### ISSUE TITLE
Some questions on training own data

### ISSUE THREAD

**Issue by 1010098686** [HuCaoFighting/Swin-Unet#27]
Hi Hu Cao:
Thanks for your great work and I am very interested in swin-unet. I use this git repo to train my own data but the predicted segmentation mask seems like composed of image patches and the border is blurry. Can you provide some suggestions? Thank you. My config is as follows:
img_size:224
depth: [2,2,2,2]
window_size:7
patch_size:4

**Comment by Pikachu-zzZ** [comment]
Hello, the prediction results I also encountered are composed of  image patches. How to improve this problem, thank you very much

**Comment by qingkuangshu** [comment]
What is your dataset format?

**Comment by Virgilzz** [comment]
怎么训练.png的数据集呢

**Comment by Flyingdog-Huang** [comment]
> 怎么训练.png的数据集呢

you need rewrite the dataloder

**Comment by Flyingdog-Huang** [comment]
> What is your dataset format?
yep, I also think the problem may be the difference between input datas' format, and zoom() func is used to resize datas, maybe this is a bigger problem

**Comment by EleFreelance** [comment]
@1010098686 我遇到跟你一摸一样的问题，你最后怎么解决的？
